WORKSPACE

```
@@ -0,0 +1,253 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+bazel_skylib_version = "1.4.1"
+
+bazel_skylib_sha = "b8a1527901774180afc798aeb28c4634bdccf19c4d98e7bdd1ce79d1fe9aaad7"
+
+http_archive(
+    name = "bazel_skylib",
+    sha256 = bazel_skylib_sha,
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/%s/bazel-skylib-%s.tar.gz" % (bazel_skylib_version, bazel_skylib_version),
+        "https://github.com/bazelbuild/bazel-skylib/releases/download/%s/bazel-skylib-%s.tar.gz" % (bazel_skylib_version, bazel_skylib_version),
+    ],
+)
+
+http_archive(
+    name = "rules_python",
+    sha256 = "5868e73107a8e85d8f323806e60cad7283f34b32163ea6ff1020cf27abef6036",
+    strip_prefix = "rules_python-0.25.0",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.25.0/rules_python-0.25.0.tar.gz",
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
+
+py_repositories()
+
+rules_scala_version = "6.1.0"
+
+rules_scala_sha = "cc590e644b2d5c6a87344af5e2c683017fdc85516d9d64b37f15d33badf2e84c"
+
+http_archive(
+    name = "io_bazel_rules_scala",
+    sha256 = rules_scala_sha,
+    strip_prefix = "rules_scala-%s" % rules_scala_version,
+    url = "https://github.com/bazelbuild/rules_scala/releases/download/v%s/rules_scala-v%s.tar.gz" % (rules_scala_version, rules_scala_version),
+)
+
+load("@io_bazel_rules_scala//:scala_config.bzl", "scala_config")
+
+# Stores Scala version and other configuration
+# 2.12 is a default version, other versions can be use by passing them explicitly:
+# scala_config(scala_version = "2.11.12")
+# Scala 3 requires extras...
+#   3.2 should be supported on master. Please note that Scala artifacts for version (3.2.2) are not defined in
+#   Rules Scala, they need to be provided by your WORKSPACE. You can use external loader like
+#   https://github.com/bazelbuild/rules_jvm_external
+scala_config(scala_version = "2.13.6")
```

The latest Scala 2.13 version is "2.13.12"

```suggestion
scala_config(scala_version = "2.13.6")
```

---

When I was writing this `rules_scala` didn't support 2.13.12, but apparently[ the newest version released last week does](https://github.com/bazelbuild/rules_scala/releases/tag/v6.2.0)